ubyssey/static/src/styles/main.scss

```diff
@@ -34,6 +34,11 @@
 @import 'components/section-page';
 @import 'components/homepage';
 
+@import 'modules/events/calendar';
+@import 'modules/events/event';
+@import 'modules/events/date_box';
+@import 'modules/events/event_box';
```

Comment:
I want to move towards using components instead of modules, so you can simply change `modules/events` to `components/events`